src/main/java/org/cactoos/collection/package-info.java

```diff
@@ -26,5 +26,8 @@
  * Collections, tests.
  *
  * @since 0.14
+ * @todo #1533:30min Continue to exploit generic variance for package
+ *  org.cactoos.collection to ensure typing works as best as
+ *  possible as it is explained in #1533 issue.
```

Comment:
@baudoliver7 I think the todo should be explicit about the fact that not only collection should be covered but all of the package. I would propose to add a similar todo into each and every package of cactoos in the current PR!

Reply:
@victornoel Ok